shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -615,4 +863,71 @@ void AccessibilityBridge::OnSemanticsModeChanged(
   delegate_.SetSemanticsEnabled(enabled);
 }
 
+void AccessibilityBridge::FillInspectTree(int32_t flutter_node_id,
```

Comment:
Let's just wrap FillInspectTree and all of the FooToString methods it uses completely in #if !FLUTTER_RELEASE

The only code that calls this is already wrapped inside of a #if

Completely wrapping this function (and the others) will remove even more bytes from the release binary, and you will not be forced to maintain "degenerate" FooToString functions that return only an empty string